src/components/ChartMode.tsx

```diff
@@ -66,17 +67,18 @@ const ChartMode = ({ onChartType, onGranularity, portalNodeId }: TChartModeProps
                     </div>
                 </div>
                 {allowTickChartTypeOnly && (
-                    <div
-                        className={classNames('sc-chart-mode__section__text', {
-                            'sc-chart-mode__section__text--mobile': isMobile,
-                        })}
-                    >
-                        {t.translate('Only selected charts and time intervals are available for this trade type.')}
-                    </div>
+                    <InfoFootnote
+                        isMobile={isMobile}
+                        text={t.translate('Only selected charts and time intervals are available for this trade type.')}
+                    />
                 )}
             </Menu.Body>
         </Menu>
     );
 };
 
+ChartMode.defaultProps = {
```

Comment:
cant we just initialize instead of doing this

Reply:
<img width="1120" alt="Screenshot 2023-10-23 at 6 28 33 PM" src="https://github.com/binary-com/SmartCharts/assets/103177211/957ecc8c-8d2c-4aaf-81b6-0182988fc581">
i tried it but it didn't help. it's about project settings.